README.md

```diff
@@ -37,10 +37,12 @@ Since webshims does not support fingerprinting, this will result in 404s (missin
    (Note that this should be run directly, not in a dom-ready block.)
 
   ```javascript
-  $.webshims.setOptions('basePath', '/assets/webshims/shims/')
+  $.webshims.setOptions('basePath', '/webshims/shims/1.15.6/')
```

Comment:
Looking at this change.. I don't think it's a good idea to hardcode the webshims version into the application. That's pretty much what we are trying to avoid in all this.

Maybe we should use the webshims_path helper here as an example.